.travis/install-macos.sh

```diff
@@ -2,3 +2,4 @@
 
 brew update
 brew install freetype cairo
```

Comment:
I noticed the macOS build is failing, and it might be fixed (and become faster) by removing the brew update & brew install statements.

I don't think we need to update brew and also don't need freetype & cairo since they are already bundled as git submodules.

It seems we only need git & cmake which should already be installed by default on Travis.

Reply:
> I don't think we need to update brew and also don't need freetype & cairo since they are already bundled as git submodules.

My position is that libraries that are available on the system should not be linked as submodules, and also should be linked dynamically, if possible at all.

Reply:
Agreed, however, cairo and freetype are not installed by default as far as I know, so I think we should make sure they are included (on macOS and windows deliverables) also, the CI should probably test the dependencies that are linked as submodules to make sure they work.

I have seen many OSS failing on macOS due to some missing library and otool workarounds (even on this repo if I remember correctly) and I guess that is not what we want.

As an added bonus, I guess the libraries that are in use are not that big at all.

Reply:
> I have seen many OSS failing on macOS due to some missing library and otool workarounds (even on this repo if I remember correctly) and I guess that is not what we want.

They're now linked statically instead, which is what I should have done in the first place.

> the CI should probably test the dependencies that are linked as submodules to make sure they work.

Our CI is already quite slow and testing the dependency version that is updated once a few months at best is a waste of time.